mantidproject/mantid
# Vates testing docs

**Description of work.**

This PR adds testing for the VATES interface. This testing will be done in each release cycle to ensure that the interface is in minimal required working condition. This means that all features that are regularly/routinely required by users should be tested and verified.

**Report to:** nobody. 

**To test:**

* Build the `dev-docs-html`
* Open `dev-docs/index.html` and navigate to `Testing` > `VATES Testing`
* Check that the tests make sense and cover all required features

*There is no associated issue.*

*This does not require release notes* because it is not a user-facing feature.
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
